resources/qml/WelcomePages/AddLocalPrinterScrollView.qml

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) 2019 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+
+import UM 1.3 as UM
+import Cura 1.0 as Cura
+
+
+//
+// This is the scroll view widget for adding a (local) printer. This scroll view shows a list view with printers
+// categorized into 3 categories: "Ultimaker", "Custom", and "Other".
+//
+ScrollView
+{
+    id: base
+
+    property var currentItem: (machineList.currentIndex >= 0)
+                              ? machineList.model.getItem(machineList.currentIndex)
+                              : null
+    property string currentSection: preferredCategory
```

Comment:
Some documentation here would be nice. WHat do these properties do?

Reply:
Added in 47d74950d